java/openjdk8-powerpc/Portfile

```diff
@@ -0,0 +1,324 @@
+                            --with-extra-cxxflags="${configure.cxxflags} -arch ${configure.build_arch}" \
+                            --with-extra-ldflags="${configure.ldflags} -I${prefix}/lib -arch ${configure.build_arch}"
+    configure.args-delete   --with-extra-cflags="${configure.cflags}" \
+                            --with-extra-cxxflags="${configure.cxxflags}" \
+                            --with-extra-ldflags="${configure.ldflags}"
+}
+
+if {![variant_isset debug] && ![variant_isset release]} {
+    default_variants-append +release
+}
+
+default_variants-append     +zero
+
+build.type          gnu
+build.target        images
+
+use_parallel_build  no
+
+set jdkn jdk1.${major}.0_${update}.jdk
+set jren jre1.${major}.0_${update}.jre
+set jdk_path ${tpath}/JavaVirtualMachines/${real_name}
+set jre_path ${tpath}/JavaVirtualMachines/${real_name}-jre
+
+if {[variant_isset zero]} {
+    # We do not build macOS-style bundles here, so paths differ:
+    set jdk_bundle_dir build/openjdk8/images/j2sdk-image
+    set jre_bundle_dir build/openjdk8/images/j2re-image
+
+    # But we want to install in a macOS-style:
+    destroot {
+        xinstall -m 755 -d ${destroot}${jdk_path}/Contents
+        xinstall -m 755 -d ${destroot}${jre_path}/Contents
+        # This renames j2*-image dirs to respective Homes:
+        move ${worksrcpath}/${jdk_bundle_dir} ${destroot}${jdk_path}/Contents/Home
+        move ${worksrcpath}/${jre_bundle_dir} ${destroot}${jre_path}/Contents/Home
+        # We need these:
+        system "ln -s ${prefix}/lib/libfontconfig.1.dylib ${destroot}${jdk_path}/Contents/Home/jre/lib/libfontconfig.dylib"
+        system "ln -s ${prefix}/lib/libfontconfig.1.dylib ${destroot}${jre_path}/Contents/Home/lib/libfontconfig.dylib"
+    }
+    # Binaries are gone from worksrcpath, so run from destroot:
+    test.cmd        ${destroot}${jdk_path}/Contents/Home/bin/java
+
+} else {
+    set jdk_bundle_dir build/openjdk8/images/j2sdk-bundle/${jdkn}/Contents
+    set jre_bundle_dir build/openjdk8/images/j2re-bundle/${jren}/Contents
+
+    destroot {
+        xinstall -m 755 -d ${destroot}${jdk_path}
+        xinstall -m 755 -d ${destroot}${jre_path}
+        copy ${worksrcpath}/${jdk_bundle_dir} ${destroot}${jdk_path}
+        copy ${worksrcpath}/${jre_bundle_dir} ${destroot}${jre_path}
+    }
+
+    test.cmd        ${worksrcpath}/${jdk_bundle_dir}/Home/bin/java
+}
+
+destroot.violate_mtree  yes
+
+test.run                yes
+test.target             -version
+
+post-destroot {
+    delete ${worksrcpath}
+}
+
+notes "
+If you want to make ${name} the default JDK, add this to shell profile:
+export JAVA_HOME=${jdk_path}/Contents/Home
+If you want to make the JRE installed by the ${real_name} the default JRE, add this to shell profile:
+export JAVA_HOME=${jre_path}/Contents/Home
+"
+
+livecheck.type      regex
+livecheck.url       https://github.com/openjdk/jdk8u/tags
```

Comment:
I'd recommend using `${master_sites}` (https://openjdk-sources.osci.io/openjdk8/) for the livecheck as well.

Reply:
I do not know how these work. Can I just replace URL?

Reply:
See the documentation for the [livecheck options](https://guide.macports.org/chunked/reference.livecheck.html).

You can set `livecheck.url` to `https://openjdk-sources.osci.io/openjdk8/`. `livecheck.regex` looks like it should keep working as-is, but you could make it slightly more specific while you're at it, like `openjdk(8u\\d+)-ga`.

Reply:
Done